composables/useNotification.ts

```diff
@@ -0,0 +1,39 @@
+type Theme = "success" | "error" | "";
+
+const theme = ref<Theme>("");
+const text = ref("");
+const isOpen = ref(false);
+const time = ref(10000);
+const timeout = ref();
+
+export const useNotification = () => {
+  const { hideScrollY } = usePreventBodyScroll();
+
+  const notify = (type: Theme, message: string) => {
+    clearTimeout(timeout.value);
+
+    isOpen.value = true;
+    text.value = message;
+    theme.value = type;
+
+    timeout.value = setTimeout(() => {
+      closeNotification();
+    }, time.value);
+  };
+
+  const closeNotification = () => {
+    isOpen.value = false;
+  };
+
+  watch(isOpen, () => {
+    hideScrollY(isOpen.value);
```

Comment:
is it needed to block scrolling if notification is opened? 🤔

Reply:
good point :), I will merge this one, and delete in the next PR where it is a bit differently organized so it will be easier